sql/_33_elderberry/cbrd_23844/cbrd_24195/answers/create1.answer

```diff
@@ -0,0 +1,46 @@
+===================================================
+0
+===================================================
+0
+===================================================
+class_of    unique_name    class_name    owner.name    
+null     u1.t1     t1     U1     
+
+===================================================
+0
+===================================================
+unique_name    name    owner.name    
+u1.s1     s1     U1     
+
+===================================================
+0
+===================================================
+0
+===================================================
+0
+===================================================
+unique_name    name    owner.name    target_class    action_definition    
+u1.trig1     trig1     U1     null     insert into [dba.t2] values ([obj].[c1])     
```

Comment:
target_class must be dba.t1.

Reply:
In CTP output, object type is displayed as null.